nixos/modules/programs/less.nix

```diff
@@ -35,7 +33,8 @@ in
 
     programs.less = {
 
-      enable = mkEnableOption "less";
+      # since environment.nix sets PAGER=less, enable the less module by default
+      enable = mkEnableOption "less" // { default = true; };
```

Comment:
Instead of defaulting to true here and relying on a comment to make the link, enable it in the environments.nix file.

Reply:
Done.